pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java

```diff
@@ -1242,27 +1277,55 @@ public ResultSet getColumns(String catalog,
       tuple[5] = sqlType.getBytes();
       tuple[6] = null;
       tuple[7] = null;
-      tuple[8] = null;
-      tuple[9] = connection.encodeString("10");
       tuple[10] = connection.encodeString(Integer.toString(columnNullable));
       tuple[11] = null;
-      tuple[12] = null;
       tuple[13] = null;
       tuple[14] = null;
-      tuple[15] = null;
       tuple[16] = rs.getBytes("ordinal_position");
-      tuple[17] = connection.encodeString("YES");
       tuple[18] = null;
       tuple[19] = null;
       tuple[20] = null;
       tuple[21] = null;
       tuple[22] = connection.encodeString("NO");
-      tuple[23] = connection.encodeString("NO");
-      tuples.add(tuple);
+
+      if (getCrateVersion().before("2.0.0")) {
```

Comment:
Do we intend to backport the information schema changes? Otherwise this probably needs to check against 2.1.0